### PR TITLE
Automated cherry pick of #67097: Ignore EIO error in unmount path

### PR DESCRIPTION
**Pull request by chakri-nelluri**
Cherry pick of #67097 on release-1.11.

#67097: Ignore EIO error in unmount path

### CODE REVIEW

**Comment by chakri-nelluri** [comment]
@saad-ali @gnufied @msau42 Please approve it for cherry-pick

**Comment by chakri-nelluri** [comment]
@ivan4th PTAL when you get a chance.

**Comment by chakri-nelluri** [comment]
@ivan4th Please merge it. This is already merged to 1.10.7 release.

**Comment by chakri-nelluri** [comment]
@screeley44 @ivan4th 1.11.3 is already out with out this fix. Please cherry pick it. 1.10.7 already has this fix merged.

File: pkg/volume/util/util.go [hunk 1]
```diff
@@ -188,15 +188,15 @@ func PathExists(path string) (bool, error) {
...
+	} else if IsCorruptedMnt(err) {
 		return true, err
 	} else {
 		return false, err
 	}
 }
 
-// isCorruptedMnt return true if err is about corrupted mount point
-func isCorruptedMnt(err error) bool {
+// IsCorruptedMnt return true if err is about corrupted mount point
```
Comment (ivan4th) [inline pkg/volume/util/util.go]: returns

**Comment by ivan4th** [comment]
There's small typo in a comment but it's a cherrypick, plus it was there before this PR.
/lgtm

**Comment by nikhita** [comment]
/assign @foxish 
for cherry-pick approval

**Pull request merged**